services/ui-src/src/components/pages/GetStarted/McparGetStartedPage.tsx

```diff
@@ -50,6 +55,30 @@ export const McparGetStartedPage = () => {
               <Text sx={sx.additionalInfo}>{section2.additionalInfo}</Text>
             </Box>
           </Flex>
+          {section2.table ? (
+            <Flex sx={sx.sectionContent}>
+              <Box>
+                <Heading sx={sx.smallSectionHeading} size={"sm"}>
+                  {section2.tableHeading}
+                </Heading>
+                <Table content={section2.table}></Table>
+              </Box>
+            </Flex>
+          ) : (
+            <></>
+          )}
+          {section2.autosaveNotice && section2.autosaveHeading ? (
+            <Flex sx={sx.sectionContent}>
+              <Box>
+                <Heading sx={sx.smallSectionHeading} size={"sm"}>
+                  {section2.autosaveHeading}
+                </Heading>
+                <Text>{section2.autosaveNotice}</Text>
+              </Box>
+            </Flex>
+          ) : (
+            <></>
+          )}
```

Comment:
Question: Do these have to use ternary ops or could they just be `&&` and not return anything if the conditions aren't true? Non-blocking.

Reply:
I tried that, but I was getting typescript errors! This was the cleanest solution I could find. Happy to add that as a fast follow.